README.md

```diff
@@ -180,68 +105,55 @@ Runtime Requirements
   * Windows 7+ for Win games
   * Modern browsers and IE 9+ for web games
 
-Running Tests
+Environment Setup
 --------------------
 
-Select the test you want from Xcode Scheme chooser.
+Should set up environment before starting a new game or running tests
 
-* Cocos Console
-
-```
-// Enter cpp test folder
-cd tests/cpp-tests
-// Or enter js test folder
-cd tests/js-tests
-// Or enter lua test folder
-cd tests/lua-tests
-
-// Compile or run test case
-cocos compile -p ios|mac|android|win32|win8_1|metro|web -m debug|release
-cocos run -p ios|mac|android|win32|win8_1|metro|web -m debug|release
 ```
-
-* For OS X / iOS
+$ cd cocos2d-x
+$ ./setup.py
+$ source FILE_TO_SAVE_SYSTEM_VARIABLE
```

Comment:
This doesn't make sense.
I have neither an environment variable called `FILE_TO_SAVE_SYSTEM_VARIABLE`, nor a file with that name.